profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -188,33 +190,37 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
         }
       }
 
-      // Provide 'Request translation' action or hide actions.
-      // NEXTEUROPA-7719 - Simplifying Poetry UI - as a part of this ticket
-      // there is new #markup element which will be displayed every time when
-      // there is a translation request on going.
-      if ($translation_possible && !$existing_jobs) {
-        $form['actions']['#type'] = 'actions';
-        $form['actions']['request'] = [
-          '#type' => 'submit',
-          '#value' => t('Request translation'),
-          '#submit' => ['tmgmt_workbench_translate_form_submit'],
-          '#validate' => ['tmgmt_workbench_translate_form_validate'],
-        ];
-      }
-      else {
-        unset($form['actions']);
-        foreach ($lang_codes as $lang_code) {
-          $form['languages'][$lang_code] = [
-            '#type' => 'checkbox',
-            '#disabled' => TRUE,
+      if (!isset($form_state['force_actions']) || !$form_state['force_actions']) {
+        // Provide 'Request translation' action or hide actions.
+        // NEXTEUROPA-7719 - Simplifying Poetry UI - as a part of this ticket
+        // there is new #markup element which will be displayed every time when
+        // there is a translation request on going.
+        if ($translation_possible && !$existing_jobs) {
+          $form['actions']['#type'] = 'actions';
+          $form['actions']['request'] = [
+            '#type' => 'submit',
+            '#value' => t('Request translation'),
+            '#submit' => ['tmgmt_workbench_translate_form_submit'],
+            '#validate' => ['tmgmt_workbench_translate_form_validate'],
           ];
         }
-        $form['languages_description'] = [
-          '#type' => 'item',
-          '#prefix' => '<div class="messages warning">',
-          '#markup' => t('Please end up the active translation process before creating a new request.'),
-          '#suffix' => '</div>',
-        ];
+        else {
+          unset($form['actions']);
+          foreach ($lang_codes as $lang_code) {
+            $form['languages'][$lang_code] = [
+              '#type' => 'checkbox',
+              '#disabled' => TRUE,
+            ];
+          }
+          if (!isset($form['languages_description'])) {
+            $form['languages_description'] = [
```

Comment:
You are right, but in this case, we need to include the message on the zone of the button "Request translation".